source/simulator/assemblers/advection.cc

```diff
@@ -468,6 +468,307 @@ namespace aspect
 
 
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Remove some of the extra empty lines above (only need three between functions) :)